### PR TITLE
Bug 1215312 - [Messages] In the Message Report, hide the lateness ind…

### PR DESCRIPTION
**Pull request by julienw**
…icator for outgoing messages r=schung

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d4d9cb42d2a0d059fccb2ee569022e83f0e72f67

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=288fbecadb8ea588ad8e6fc0e14c40000d8e87a5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=288fbecadb8ea588ad8e6fc0e14c40000d8e87a5

File: apps/sms/views/conversation/test/unit/information_test.js [hunk 1]
```diff
@@ -307,16 +307,35 @@ suite('Information view', function() {
...
+        sentTitle,
+        contactTitle,
+        sizeHide,
+        sizeContent,
+        isIncoming
+      } = opts;
 
       assert.equal(reportView.container.dataset.delivery, delivery);
+      if (isIncoming) {
+        assert.isTrue(
```
Comment (steveck-chung) [inline apps/sms/views/conversation/test/unit/information_test.js]: nit: maybe 
```
        assert.equal(
          reportView.container.classList.contains('incoming-message'), isIncoming
        );
        assert.equal(
          reportView.container.classList.contains('outgoing-message'), !isIncoming
        );
```  ?

Just few lines less, not really important.

Comment (julienw) [inline apps/sms/views/conversation/test/unit/information_test.js]: ah yeah you're right

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ffc1fc2a4d8bc39bd31eed437c2af4b828d8ae00

**Pull request merged**